core-services/prow/02_config/windup/windup-ui-tests/_pluginconfig.yaml

```diff
@@ -0,0 +1,4 @@
+plugins:
+  windup/windup-ui-tests:
```

Comment:
This is just a list of plugins for the [OpenShift CI bots](https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/#granting-robots-privileges-and-installing-the-github-app) to use in the windup/windup-ui-tests repository. I just did the bare minimum here because that team is only using OpenShift CI for this test right now.

Reply:
I see. Thanks!